File: StykerDG.FarmForge.DataModel/Models/UnitType.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace StrykerDG.FarmForge.DataModel.Models
{
    [Table("UnitType")]
    public class UnitType : BaseModel
    {
        [Key]
        public int UnitTypeId { get; set; }
        public string Name { get; set; }
        public string Label { get; set; }
        public string Description { get; set; }
    }
}

File: StrykerDG.FarmForge.Client/farmforge_client/lib/models/inventory/unit_type.dart
class UnitType {
  int unitTypeId;
  String name;
  String label;
  String description;

  UnitType({
    this.unitTypeId = 0,
    this.name,
    this.label,
    this.description
  });

  factory UnitType.fromMap(Map<String, dynamic> data) {
    return UnitType(
      unitTypeId: data['UnitTypeId'],
      name: data['Name'],
      label: data['Label'],
      description: data['Description']
    );
  }

  Map<String, dynamic> toMap() {
    return {
      'UnitTypeId': this.unitTypeId,
      'Name': this.name,
      'Label': this.label,
      'Description': this.description
    };
  }
}